packages/utilities/gel-foundations-styled-components/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@bbc/gel-foundations-styled-components",
+  "version": "0.1.0",
+  "description": "A collection of gel styled components",
```

Comment:
I think our package descriptions should be more descriptive than this. What is GEL? Sure, this contains styled components, but what of?